package poo.excepciones;

public class Test {
	
	public int metodoSeguro() throws Exception{
		
		return 1;
		
	}

}
